dataproxy/cloudflare-workers-module/package.json

```diff
@@ -5,7 +5,8 @@
     "test": "jest"
   },
   "dependencies": {
-    "@prisma/client": "4.17.0-dev.77"
+    "@prisma/client": "4.17.0-dev.77",
+    "@prisma/extension-accelerate": "0.3.1"
```

Comment:
How do we go about updating this? Renovate?

Reply:
Renovate would be good, I think for this package.